mesheryctl/internal/cli/root/system/dashboard.go

```diff
@@ -0,0 +1,100 @@
+// Copyright 2020 Layer5, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package system
+
+import (
+	"os/exec"
+
+	"github.com/pkg/errors"
+
+	"github.com/layer5io/meshery/mesheryctl/internal/cli/root/config"
+	"github.com/layer5io/meshery/mesheryctl/pkg/utils"
+
+	"net/http"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+// dashboardCmd represents the - command
+var dashboardCmd = &cobra.Command{
+	Use:   "dashboard",
+	Short: "Open Meshery UI",
+	Long:  `Well we might change it later`,
+	Args:  cobra.NoArgs,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		if err := dashboard(); err != nil {
+			return errors.Wrap(err, "failed to open the Meshery UI")
+
+		}
+		return nil
+
+	},
+}
+
+func dashboard() error {
+
+	// Get viper instance used for context
+	mctlCfg, err := config.GetMesheryCtl(viper.GetViper())
```

Comment:
We provide the reformat tools under the project, https://github.com/meshery/meshery/blob/1ff876f10e0a9a0387bb84f96ca0a2edb1add6cd/Makefile#L90-L92 , please kindly check how to use this tool under localhost.

Reply:
If this does not work for you, please modify the environment `GO111MODULE` and `GOPROXY`